packages/arb-token-bridge-ui/src/generateOpenGraphImages.tsx

```diff
@@ -0,0 +1,345 @@
+import React from 'react'
+import satori, { Font } from 'satori'
+import sharp from 'sharp'
+import fs from 'fs'
+import { ChainId, isNetwork } from './util/networks'
+import { getBridgeUiConfigForChain } from './util/bridgeUiConfig'
+import { orbitMainnets, orbitTestnets } from './util/orbitChainsList'
+
+const dimensions = {
+  width: 1200,
+  height: 627
+} as const
+
+async function getFonts(): Promise<Font[]> {
+  const unica77_regular = fs.readFileSync('./src/font/Unica77LL-Regular.otf')
+  const unica77_medium = fs.readFileSync('./src/font/Unica77LL-Medium.otf')
+
+  return [
+    {
+      name: 'Unica77',
+      data: unica77_regular,
+      weight: 400,
+      style: 'normal'
+    },
+    {
+      name: 'Unica77',
+      data: unica77_medium,
+      weight: 500,
+      style: 'normal'
+    }
+  ]
+}
+
+type Chain = ChainId | number
+
+type ChainCombination = [Chain, Chain]
+
+const chainToLogo: { [key: Chain]: string } = {
+  [ChainId.Ethereum]: 'https://l2beat.com/icons/ethereum.png',
+  [ChainId.ArbitrumOne]: 'https://l2beat.com/icons/arbitrum.png',
+  [ChainId.ArbitrumNova]: 'https://l2beat.com/icons/nova.png'
+}
+
+const configs: ChainCombination[] = [
+  [ChainId.Ethereum, ChainId.ArbitrumOne],
+  [ChainId.Ethereum, ChainId.ArbitrumNova],
+  [ChainId.ArbitrumOne, 660279],
+  ...Object.values(orbitMainnets).map(
+    chain => [chain.parentChainId, chain.chainId] as ChainCombination
+  ),
+  ...Object.values(orbitTestnets).map(
+    chain => [chain.parentChainId, chain.chainId] as ChainCombination
+  )
+]
+
+function ChainWrapper({
+  chain,
+  direction
+}: {
+  chain: Chain
+  direction: 'From' | 'To'
+}) {
+  const chainConfig = getBridgeUiConfigForChain(chain)
+  return (
+    <div
+      style={{
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        width: '760px',
+        gap: '6px',
+        padding: '25px',
+        backgroundColor: `${chainConfig.color}40`,
+        border: '1px solid',
+        borderColor: chainConfig.color,
+        borderRadius: '5px'
+      }}
+    >
+      <span
+        style={{
+          display: 'block',
+          color: 'white',
+          backgroundColor: chainConfig.color,
+          padding: '15px 20px',
+          fontSize: '60px',
+          borderRadius: '5px'
+        }}
+      >
```

Comment:
We can use tailwind here, and just override backgroundColor with style

Reply:
satori has crappy tailwind support
![image](https://github.com/user-attachments/assets/83729d65-d71f-4fda-a61d-cf63b14d418b)
![image](https://github.com/user-attachments/assets/c88e6efd-3977-4e04-bdc9-e1b4db9d9adf)
i’d rather not use tailwind and use the style object directly, rather than testing which ones they don’t support correctly and use 50% tw 50% style object…